meta-balena-common/recipes-core/images/resin-image.bb

```diff
@@ -24,7 +24,6 @@ inherit core-image image-resin distro_features_check
 IMAGE_FEATURES_append = " \
```

Comment:
Typo in commit log: `We have not explicitly asked Yocto to install dropbear`. I believe the `not` should be `now` instead?